File: tests/fixtures/projects/python-demo/src/math.py
"""纯函数：covtrim python 封装的覆盖率目标（div 含 if 分支未测 → 部分覆盖）。"""


def add(a, b):
    return a + b


def mul(a, b):
    return a * b


def div(a, b):
    if b == 0:
        raise ValueError("div by zero")
    return a / b

File: tests/fixtures/projects/deno-demo/src/math.ts
/** 纯函数：covtrim deno 封装的覆盖率目标（div 含分支未测 → 部分覆盖）。 */

export function add(a: number, b: number): number {
  return a + b;
}

export function mul(a: number, b: number): number {
  return a * b;
}

export function div(a: number, b: number): number {
  if (b === 0) throw new Error('div by zero');
  return a / b;
}
